README.md

```diff
@@ -104,12 +105,43 @@ evaluator.evaluate()
 risk = evaluator.risk()
 ```
 
+## Configuring logging
+
+`Anonymeter` uses the standard Python logger namespaced under `anonymeter`.
+It inherits the logging configuration from the root logger,
+but you can override it to suit your needs.
+
+
+For example, to set the logging level to `DEBUG` you can use the following snippet:
+
+```python
+import logging
+
+# set the logging level to DEBUG
+logging.getLogger("anonymeter").setLevel(logging.DEBUG)
+```
+
+And if you want to log to a file, you can use the following snippet:
+
+```python
+import logging
+
+# create a file handler
+file_handler = logging.FileHandler("anonymeter.log")
+
+# set the logging level for the file handler
+file_handler.setLevel(logging.DEBUG)
+
+# add the file handler to the logger
+logging.getLogger("anonymeter").addHandler(file_handler)
```

Comment:
```suggestion
logging.getLogger("anonymeter").addHandler(file_handler).setLevel(logging.DEBUG)
```
If I'm not mistaken, the default root logger level is `WARNING` and nothing will be written to the `DEBUG` handler. I understand that this is just an example, but 
it makes it more workable.